storage/tianmu/handler/tianmu_handler.cpp

```diff
@@ -1474,11 +1474,18 @@ int TianmuHandler::reset() {
 
 enum_alter_inplace_result TianmuHandler::check_if_supported_inplace_alter([[maybe_unused]] TABLE *altered_table,
                                                                            Alter_inplace_info *ha_alter_info) {
+  DBUG_ENTER(__PRETTY_FUNCTION__);
   if ((ha_alter_info->handler_flags & ~TIANMU_SUPPORTED_ALTER_ADD_DROP_ORDER) &&
       (ha_alter_info->handler_flags != TIANMU_SUPPORTED_ALTER_COLUMN_NAME)) {
```

Comment:
return the code flow as soon as possible.

if (xxx) --not satisfied
   return xxxx;
if (yyy)
   return yyyy

 ....